### DISCUSSION TITLE
How to start document_retagger in Docker?

### DISCUSSION THREAD

**Discussion by felix-wa** [jonaswinkler/paperless-ng#1616] [Support] [answered]
Hello, 
I am quite new to paperless and try to figure out how to start the document_retagger. I am running paperless-ng in a Docker container on a synology NAS. I am able to get in the terminal but do not know how to continue here. in which folder is the script I need to call how how should I call it?
Best
F

**Comment by sbrunner** [comment, marked as answer]
`docker-compose exec webserver python3 manage.py document_retagger` :-)

**Reply by felix-wa** [reply]
Hey, thanks for your fast reply. I also found the command in my online research, but I have no idea where to enter it. In my Synology I am loggen on to the container as root and see different folders and files, I can not run it here. I also have not found any place in the webUI where I can enter it.
Can you help out here?

**Reply by sbrunner** [reply]
From the container just do `python3 manage.py document_retagger`

From pout of the container `docker exec <containername> python3 manage.py document_retagger` :-)

**Reply by felix-wa** [reply]
hmm.. I can start python3 within the container but when trying to start manage.py the answer is that 'No such file or directory' exists. Is there some documentation in which folder I can find the manage.py file?

**Reply by sbrunner** [reply]
He is in the folder `/usr/src/paperless/src` :-)

**Reply by felix-wa** [reply]
hello @sbrunner, thanks again for your fast reply.
The folder you are referring at does not exist for me, I can navigate to `/usr/src/` but there is nothing in this container any more.

**Reply by felix-wa** [reply]
hello again.. I clicked to fast :-)
The mentioned folder does not exist for me but I found the file in `/app/paperless/src`. From there I can call `python3 manage.py document_retagger`.

Thanks for your replies to my post 👍
